docs-app/package.json

```diff
@@ -108,7 +108,7 @@
     "postcss": "^8.4.17",
     "postcss-import": "^15.0.0",
     "postcss-loader": "^7.0.1",
-    "prettier": "^2.8.3",
+    "prettier": "^3.0.1",
```

Comment:
Needed to update prettier so that the prettier template plugin would work properly